tests/installation/yast_fullscreen.pm

```diff
@@ -0,0 +1,20 @@
+# SUSE's openQA tests
+#
+# Copyright 2022 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: Enlarge the YaST window for fullscreen in ssh-X test.
+
+# Maintainer: QA SLE YaST team <qa-sle-yast@suse.de>
+
+use strict;
+use warnings;
+use base 'y2_installbase';
+use testapi;
```

Comment:
not needed.

Reply:
Tell this to the CI. ;-) 
Started without this and got complaints...

Reply:
mmm perhaps the difference between `use parent` and `use base`, strange, you're right.